tests/integration/multi-arch.bash

```diff
@@ -2,10 +2,10 @@
 get_busybox() {
 	case $(go env GOARCH) in
 	arm64)
-		echo 'https://github.com/docker-library/busybox/raw/dist-arm64v8/glibc/busybox.tar.xz'
+		echo 'https://github.com/docker-library/busybox/raw/dist-arm64v8/stable/glibc/busybox.tar.xz'
```

Comment:
Wouldn't it be more stable to use a specific commit here and document how/when to update it? :innocent: :grimacing:

Reply:
We always used the latest version, and I never saw an issue. Once we'll switch to a specific version, it will never be updated :(

Your point is valid though. Let me see.

Reply:
@tianon updated; PTAL

Reply:
LGTM; busybox should be pretty stable (and limited scope here), so I doubt it really *needs* to be updated very often :sweat_smile:

Reply:
OK I am moving all the improvements to a separate PR, as this one we need to merge ASAP to fix CI.